Comp2023/src/main/java/frc/robot/Robot.java

```diff
@@ -41,6 +42,18 @@ public void robotInit( )
     DataLogManager.start( );
     DataLogManager.log("RobotInit: RoboRIO SN:" + System.getenv("serialnum"));
 
+    if (System.getenv("serialnum").equals("03238074"))
+    {
+      Constants.isComp = true;
+      DataLogManager.log("COMP BOT!");

```

Comment:
I really like the log message--maybe a little more context "Detected the COMPETITION (A) robot!" and Detected the PRACTICE (B) robot!".